doc/content/reference/configuration/packet-broker-agent.md

```diff
@@ -3,37 +3,42 @@ title: "Packet Broker Agent Options"
 description: ""
 ---
 
+## Registration Options {{< new-in-version "3.12.0" >}}
+
+- `pba.registration.name`: Friendly name to register with Packet Broker
+- `pba.registration.administrative-contact.email`: Email address of the administrative contact person or mailing group
+- `pba.registration.technical-contact.email`: Email address of the technical contact person or mailing group
+- `pba.registration.listed`: Indicates whether the Home Network is listed in the Packet Broker catalog. Set this to `false` to connect to Packet Broker but to stay private to other networks
+
 ## Connection Options
 
-- `pba.data-plane-address`: Address of Packet Broker Data Plane
+- `pba.iam-address`: Address of Packet Broker IAM {{< new-in-version "3.12.0" >}}
+- `pba.control-plane-address`: Address of Packet Broker Control Plane {{< new-in-version "3.12.0" >}}
+- `pba.data-plane-address`: Address of Packet Broker Data Plane. See [Packet Broker Clients](https://github.com/packetbroker/pb) for available cluster addresses
+- `pba.insecure`: Connect without using TLS (only for test environments)
 - `pba.net-id`: LoRa Alliance NetID
 - `pba.tenant-id`: Tenant ID within the NetID
 - `pba.cluster-id`: Cluster ID uniquely identifying this cluster within a NetID and tenant. The cluster ID is used for shared subscriptions (i.e. splitting traffic over multiple Packet Broker Agents) and as Forwarder ID to route downlink traffic to the right cluster
+- `pba.home-network-cluster-id`: Home Network Cluster ID, if different from the Cluster ID. Leave empty to fallback to `cluster-id`
 
-## Client TLS Options
```

Comment:
mTLS is not supported by Packet Broker anymore, so we don't have to mark this deprecated or anything